getting-started/scaling-out.md

```diff
@@ -0,0 +1,181 @@
+# Scaling out using Distributed Hypertables
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
+Hypertables are used to handle large amount of data by breaking it up
+into smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+Data nodes together with an *access node* constitute a distributed database
+([architecture][]). All the nodes are TimescaleDB instances,
+i.e., hosts with a running PostgreSQL database and loaded TimescaleDB extension.
+While the data nodes store distributed chunks, the access node is
+the entry point for clients to access distributed hypertables.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database on an access node
+using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and connect to an
+  existing PostgreSQL instance.
+
+* You should already have a running PostgreSQL server on the data node host.
+
+When creating the data node, you should:
+
+* Provide a name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+  
+* Provide the remote password, which will be used during access to
+  the created remote data node by the current user.
+
+* Provide a bootstrap user and password, which is used to
+  create the data node. If the current user can be used, then 
+  the boostrap user and password can be omitted.
+  
+* Ensure that the data node has password authentication enabled 
```

Comment:
Should the steps be laid out to update the `pg_hba.conf` file?

Reply:
Good catch. Moved it up into prerequisites.